Android/Samples/libraries/kml/src/main/java/com/microsoft/maps/kml/KMLParser.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package com.microsoft.maps.kml;
+
+import androidx.annotation.NonNull;
+import com.microsoft.maps.MapElementLayer;
+import com.microsoft.maps.MapIcon;
+import com.microsoft.maps.MapPolygon;
+import com.microsoft.maps.MapPolyline;
+import com.microsoft.maps.modulestools.MapFactories;
+
+/**
+ * Class that parses KML and returns a new MapElementLayer containing all the shapes outlined in the
+ * GeoJSON.
+ *
+ * <p>Created by Elizabeth Bartusiak (t-elbart) on 07/21/2020
+ */
+public class KMLParser {
+
+  private MapElementLayer mLayer;
+
+  private static final MapFactories DEFAULT_MAP_FACTORIES =
```

Comment:
Can this be on MapFactories instead so the default map factories isn't copy pasted into the two modules?